src/v/cluster/node/local_monitor.h

```diff
@@ -83,7 +82,6 @@ class local_monitor {
       std::chrono::hours(1));
     config::binding<size_t> _free_bytes_alert_threshold;
     config::binding<unsigned> _free_percent_alert_threshold;
-    config::binding<size_t> _min_free_bytes;
```

Comment:
Ran into this when chasing down an answer to a question on slack, this field is never used. Maybe a bug?

Reply:
looks ok. the configuration does seem to be used, just not via the binding which looks like it was just an oversight in the original code from aaron.

https://github.com/redpanda-data/redpanda/blob/dev/src/v/cluster/node/local_monitor.cc#L119C53-L119C53

Reply:
Yeah ultimately that was my conclusion as well.